app/dashboard/tokens.py

```diff
@@ -15,185 +15,15 @@
-    {"addr": "0x07d9e49ea402194bf48a8276dafb16e4ed633317", "name": "DALC", "decimals": 8},
-    {"addr": "0xcc4ef9eeaf656ac1a2ab886743e98e97e090ed38", "name": "DDF", "decimals": 18},
-    {"addr": "0xef68e7c694f40c8202821edf525de3782458639f", "name": "LRC", "decimals": 18},
-    {"addr": "0x3d1ba9be9f66b8ee101911bc36d3fb562eac2244", "name": "RVT", "decimals": 18},
-    {"addr": "0x8a187d5285d316bcbc9adafc08b51d70a0d8e000", "name": "SIFT", "decimals": 0},
-    {"addr": "0x8effd494eb698cc399af6231fccd39e08fd20b15", "name": "PIX", "decimals": 0},
-    {"addr": "0xaa26b73bfdc80b5c7d2cfbfc30930038fb7fa657", "name": "TOV", "decimals": 0},
-    {"addr": "0x08f5a9235b08173b7569f83645d2c7fb55e8ccd8", "name": "TNT", "decimals": 8},
-    {"addr": "0x96a65609a7b84e8842732deb08f56c3e21ac6f8a", "name": "CTR", "decimals": 18},
-    {"addr": "0xe6923e9b56db1eed1c9f430ea761da7565e260fe", "name": "FC", "decimals": 2},
-    {"addr": "0xd850942ef8811f2a866692a623011bde52a462c1", "name": "VEN", "decimals": 18},
-    {"addr": "0x2160e6c0ae8ca7d62fe1f57fc049f8363283ff5f", "name": "BPT", "decimals": 18},
-    {"addr": "0xf05a9382a4c3f29e2784502754293d88b835109c", "name": "REX", "decimals": 18},
-    {"addr": "0x73dd069c299a5d691e9836243bcaec9c8c1d8734", "name": "BTE", "decimals": 8},
-    {"addr": "0x1bcbc54166f6ba149934870b60506199b6c9db6d", "name": "ROC", "decimals": 10},
-    {"addr": "0x27f706edde3ad952ef647dd67e24e38cd0803dd6", "name": "UET", "decimals": 18},
-    {"addr": "0x0f5d2fb29fb7d3cfee444a200298f468908cc942", "name": "MANA", "decimals": 18},
-    {"addr": "0xc0e4f45b828aa4aa628e897e5da38d9dc72c2257", "name": "KC", "decimals": 8},
-    {"addr": "0x4e260e3ca268e40133c84b142de73108a7c1ec99", "name": "YC", "decimals": 0},
-    {"addr": "0xaf4dce16da2877f8c9e00544c93b62ac40631f16", "name": "MTH", "decimals": 5},
-    {"addr": "0x9214ec02cb71cba0ada6896b8da260736a67ab10", "name": "REAL", "decimals": 18},
-    {"addr": "0xe5a7c12972f3bbfe70ed29521c8949b8af6a0970", "name": "BLX", "decimals": 18},
-    {"addr": "0x5cf4e9dfd975c52aa523fb5945a12235624923dc", "name": "MPRM", "decimals": 0},
-    {"addr": "0xb581e3a7db80fbaa821ab39342e9cbfd2ce33c23", "name": "ARCD", "decimals": 18},
-    {"addr": "0x255aa6df07540cb5d3d297f0d0d4d84cb52bc8e6", "name": "RDN", "decimals": 18},
-    {"addr": "0x056017c55aE7AE32d12AeF7C679dF83A85ca75Ff", "name": "WYV", "decimals": 18},
-    {"addr": "0x89d24a6b4ccb1b6faa2625fe562bdd9a23260359", "name": "DAI", "decimals": 18},
-    {"addr": "0xfa6f7881E52fDF912c4a285D78a3141B089cE859", "name": "AVO", "decimals": 18},
-    {"addr": "0x58b6a8a3302369daec383334672404ee733ab239", "name": "LPT", "decimals": 18},
-    {"addr": "0x09617f6fd6cf8a71278ec86e23bbab29c04353a7", "name": "ULT", "decimals": 18},
-    {"addr": "0x4CEdA7906a5Ed2179785Cd3A40A69ee8bc99C466", "name": "AION", "decimals": 8},
-    {"addr": "0x4162178B78D6985480A308B2190EE5517460406D", "name": "CLN", "decimals": 18},
 
-]
+
+def get_tokens():
+    from economy.models import Token
```

Comment:
Is this imported here to prevent infinite loop in import?

Reply:
yes